src/main/java/io/jenkins/security/csp/Scanner.java

```diff
@@ -20,14 +20,15 @@
+    private static final String JS_EVENT_ATTRIBUTES = "(on(abort|afterprint|animationcancel|animationend|animationiteration|animationstart|appinstalled|auxclick|beforeinput|beforeinstallprompt|beforematch|beforeprint|beforetoggle|beforeunload|blur|cancel|canplay|canplaythrough|change|click|close|compositionend|compositionstart|compositionupdate|contentvisibilityautostatechange|contextmenu|copy|cuechange|cut|dblclick|devicemotion|deviceorientation|deviceorientationabsolute|DOMContentLoaded|drag|dragend|dragenter|dragleave|dragover|dragstart|drop|durationchange|emptied|encrypted|ended|endEvent|error|focus|focusin|focusout|formdata|fullscreenchange|fullscreenerror|gamepadconnected|gamepaddisconnected|gotpointercapture|hashchange|input|invalid|keydown|keypress|keyup|languagechange|load|loadeddata|loadedmetadata|loadend|loadstart|lostpointercapture|message|messageerror|mousedown|mouseenter|mouseleave|mousemove|mouseout|mouseover|mouseup|mousewheel|offline|online|pagehide|pageshow|paste|pause|play|playing|pointercancel|pointerdown|pointerenter|pointerleave|pointerlockchange|pointerlockerror|pointermove|pointerout|pointerover|pointerup|popstate|progress|ratechange|readystatechange|rejectionhandled|resize|resourcetimingbufferfull|resume|scroll|scrollend|securitypolicyviolation|seeked|seeking|select|selectionchange|slotchange|stalled|start|stop|storage|submit|suspend|timeupdate|timeout|toggle|touchcancel|touchend|touchmove|touchstart|transitioncancel|transitionend|transitionrun|transitionstart|unhandledrejection|visibilitychange|volumechange|waiting|wheel))";
 
     /**
      * Patterns identified in .jelly files
      */
     protected static final Map<String, Pattern> JELLY_PATTERNS = Map.of("Inline Event Handler", Pattern.compile("<[^>]+\\s" + JS_EVENT_ATTRIBUTES + "=[^>]+>", Pattern.CASE_INSENSITIVE),
             "Inline Script Block", Pattern.compile("(<script>|<script(|\\s[^>]*)[^/]>)\\s*?(?!</script>)\\S.*?</script>", Pattern.DOTALL | Pattern.CASE_INSENSITIVE),
-            "Legacy checkUrl", Pattern.compile("(checkUrl=\"[^\"]*'[^\"]*'[^\"]*\")|(checkUrl='[^']*\"[^']*\"[^']*')", Pattern.CASE_INSENSITIVE));
+            "Legacy checkUrl", Pattern.compile("(checkUrl=\"[^\"]*'[^\"]*'[^\"]*\")|(checkUrl='[^']*\"[^']*\"[^']*')", Pattern.CASE_INSENSITIVE),
+            "Javascript scheme", Pattern.compile("<[^>]+javascript:[^>]+>", Pattern.CASE_INSENSITIVE));
```

Comment:
Weird pattern. This is what I'd expect to see for an attribute, not a value. Why did you go for this?
Also
```suggestion
            "JavaScript scheme", Pattern.compile("<[^>]+javascript:[^>]+>", Pattern.CASE_INSENSITIVE));
```

Reply:
> Why did you go for this pattern?

As for java, to simply have the full tag for context purpose.

 > This is what I'd expect to see for an attribute, not a value, where I'd expect a = with optional ["' ]* before the javascript:
 
 Well I don't think it was necessary, it seems unlikely to have false positives with `javascript:` only